app/services/solution.rb

```diff
@@ -0,0 +1,38 @@
+class Solution
+
+  def initialize(solution, users, position)
+    @solution = solution
+    @users = users
+    @position = position
+  end
+  attr_reader :solution, :position
+
+  def id
+    solution[:entry_id]
+  end
+
+  def score
```

Comment:
Minor, but you can refactor (score, script, comments, created_at) as dynamically defined methods. 

```[x,y,z].each { define_method(...)}```

Reply:
For me it is not a refactor, it is a code golf . And as somebody told one day 'I do greppable code'. I think it s better to keep 'def something' .